flexget/components/sites/sites/iptorrents.py

```diff
@@ -133,6 +146,12 @@ def search(self, task, entry, config=None):
 
         entries = set()
 
+        if task.requests.domain_limiters.get(DOMAIN, None) is None:
```

Comment:
Could probably be simplified to `if not task.requests.domain_limiters.get(DOMAIN):`

Reply:
Done.